docs/_future/manifest_workflows.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Manifest workflows
+order: 4
+slug: manifest-workflows
+lang: en
+layout: future
+---
+
+<div class="section-content">
+
+Your app can use functions by referencing them in workflows. Your functions and the ones provided by slack can be used as steps in workflow definitions.
+
+Workflows are invoked by <a href="https://api.slack.com/future/triggers" target="_blank">triggers</a>. You will need to set up a trigger in order to use your defined workflows.
+
+trigger → workflow → step → function
+
+`manifest.json` is where you define your workflows.
+
+<table id="workflows">
+  <tr>
+    <th>
+      <h5>workflows</h5>
+    </th>
+    <th>
+      <code>dictionary</code>
+    </th>
+    <th></th>
+  </tr>
+  <tr>
+    <td>
+      <code>key</code>
+    </td>
+    <td>
+      <code>string</code>
+    </td>
+    <td>defines the workflows <code>id</code></td>
```

Comment:
What do you think about removing code formatting only from the tables for better readability? I'm finding the red in this table format difficult to parse! 

![Screen Shot 2022-11-02 at 2 41 33 PM](https://user-images.githubusercontent.com/55667998/199608097-43e55a9c-7538-40dc-be11-42f13d1cb52d.png)

Reply:
Yess this sounds good ill try this out

Reply:
This looks much better I think
<img width="1862" alt="Screen Shot 2022-11-03 at 3 25 20 PM" src="https://user-images.githubusercontent.com/25348381/199815709-14cdc556-2995-40ed-9bba-d96e44ae5217.png">